### ISSUE TITLE
 shape:document color is overflow

### ISSUE THREAD

**Issue by bo-ku-ra** [terrastruct/d2#533]
https://play.d2lang.com/?layout=dagre&script=yi9LLUrLyS-3UqgGu9hKISU_uTQ3Na-klgsQAAD__w%3D%3D&sketch=1&theme=103

**Comment by vfosnar** [comment]
Providing a PNG as the link is broken.

![d2](https://user-images.githubusercontent.com/13472046/219954683-f02cc9aa-0852-4bad-8514-c5b4dcf2eb6e.png)

The actual code to reproduce it should be
```d2
overflow: {shape: document}
```

**Comment by alixander** [comment]
yeah sorry, previous playground links broke with D2 0.2. thank you
